dbt/adapters/databricks/connections.py

```diff
@@ -475,6 +480,84 @@ class DatabricksConnectionManager(SparkConnectionManager):
     TYPE: str = "databricks"
     credentials_provider: Optional[TCredentialProvider] = None
 
+    def cancel_open(self) -> List[str]:
+        run_id_dir = f"/Shared/dbt_python_model/{self.profile.credentials.database}/{self.profile.credentials.schema}/run_ids/"
+        run_ids = self._list_files(run_id_dir)
+        print(run_ids)
+        for run_id in run_ids:
+            self._cancel_run_id(run_id_dir, run_id)
+
+        return super().cancel_open()
+
+    def _cancel_run_id(self, run_id_dir: str, run_id: str) -> None:
```

Comment:
since neither of these methods rely on anything in self, I think I would prefer them as static functions in python_submissions.py, so they are closer to the code that they are cleaning up.